### DISCUSSION TITLE
Looking for a Maintainer

### DISCUSSION THREAD

**Discussion by lholden** [lholden/job_scheduler#30] [Announcements]
I am currently looking for someone to take over this project. 

I haven't needed this crate for a few years now, so I haven't really had the time to keep it up to date and continue making improvements. 

If someone would like to take over ownership, please let me know!

**Comment by BlackDex** [comment]
Hello @lholden , I have already created a new crate called `job_scheduler_ng` which is a fork from this one, https://crates.io/crates/job_scheduler_ng .

It seems other people are already using my crate also. Maybe you could mention it in the readme and on crates.io maybe?
